modules/srvrlss-using-cluster-logging-find-logs-services-deployed.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// cluster-logging-serverless.adoc
+
+[id="using-cluster-logging-to-find-logs-for-services-deployed-with-Knative-Serving_{context}"]
+= Using Cluster Logging to find logs for services deployed with Knative Serving
+
+With Openshift Cluster Logging, the logs that your applications write to the console are collected in Elasticsearch. The following procedure outlines how to apply these capabilities to applications deployed by using Knative Serving.
+
+.Prerequisites
+
+* xref:../efk-logging-about_{context}[Cluster Logging components] are installed.
```

Comment:
I would suggest adding the link to appropriate assembly the additional resources section of the assembly this module will build in since we can't link module to module.